mode-lambda/static.rkt

```diff
@@ -141,6 +143,22 @@
     (hash-set! spr->idx spr pi)
     (vector-set! idx->w*h*tx*ty pi (vector w h tx ty)))
 
+
+  ; convert to premultiplied alpha
+  (for [(i (in-range (/ (bytes-length atlas-bs) 4)))]
+    (define idx (* 4 i))
```

Comment:
`([idx (in-range 0 (bytes-length atlas-bs) 4)])`

Reply:
doh - of course.  Thanks!